### PR TITLE
fix(seed): Fixed issue with jquery test for issue number 16611

### PR DESCRIPTION
**Pull request by ninadingole**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16611 

#### Description

Added some correction to the test case which was failing for an empty space or newline char in the user code.

### CODE REVIEW

**Comment by ninadingole** [comment]
fix for #16611

File: seed/challenges/03-front-end-libraries/jquery.json [hunk 1]
```diff
@@ -168,7 +168,8 @@
       ],
       "tests": [
         "assert($(\"button\").hasClass(\"animated\") && $(\"button\").hasClass(\"bounce\"), 'message: Use the jQuery <code>addClass&#40&#41</code> function to give the classes <code>animated</code> and <code>bounce</code> to your <code>button</code> elements.');",
-        "assert(!code.match(/class.*animated/g), 'message: Only use jQuery to add these colors to the element.');"
+        "assert(!code.match(/class.*animated/g), 'message: Only use jQuery to add these colors to the element.');",
+        "assert(code.match(/\\$\\(document\\)\\.ready\\(function.*(\\s|\\n)*.*button.*.addClass.*\\);/g), 'message: It is better practice to include all your jQuery code within <code>$(document).ready();</code>');"
```
Comment (Bouncey) [inline seed/challenges/03-front-end-libraries/jquery.json]: ~It is better practice to include all your jQuery code within <code>$(document).ready();</code>~

Your jQuery code should be within the <code>$(document).ready();</code> function

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: seed/challenges/03-front-end-libraries/jquery.json (hunk 1 above)
Comment (ninadingole) [reply seed/challenges/03-front-end-libraries/jquery.json]: ok will do the changes

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
Thanks for the patch @ninadingole :+1:

**Pull request merged**